client/router.js

```diff
@@ -444,7 +438,8 @@ class Router {
     this._page.base(this._basePath);
     this._page({
```

Comment:
I still believe we should pass `options.page` *Object* with all options for `page` package. Like:
```js
const pageOptions = Object.assign({
  hashbang: !!options.hashbang,
  decodeURLComponents: true
}, options.page);

this._page(pageOptions);
```

Reply:
can do that.